engines/bastion_katello/app/assets/javascripts/bastion_katello/environments/details/views/environment-details.html

```diff
@@ -12,6 +12,12 @@ <h4 translate>Basic Information</h4>
       <dt translate>Label</dt>
       <dd>{{ environment.label }}</dd>
 
+      <dt translate>Registry Name Pattern</dt>
+      <dd bst-edit-textarea="environment.registry_name_pattern"
+          on-save="save(environment)"
+          readonly="denied('edit_lifecycle_environments', environment)">
+      </dd>
```

Comment:
@waldenraines - Is there a patternfly way to add some formatted help text to a field? I want to give brief description of valid formats but it should be formatted and not just a big unreadable string.

Reply:
we do this on create, but i couldn't find any examples of on edit :(